pantalaimon/main.py

```diff
@@ -128,15 +128,17 @@ def find_proxy_by_user(user):
     default=None,
 )
 @click.option("-c", "--config", type=click.Path(exists=True))
+@click.option("--data-path", type=click.Path(exists=True))
```

Comment:
This is already possible by overriding the XDG environment variables but no harm in adding this. The man page needs to be updated to reflect this though.

Reply:
I spent about 2 days trying to figure out how to override it and gave up. If there's a better way, please do document it.

Reply:
Setting the environment variable `XDG_DATA_HOME="/home/poljar/data"` works on my box. But you are right, it might be tricky under some other environments to set it.

As I said. I'm fine with this one.

Reply:
This still needs a man page update. If you're unfamiliar with mdoc I can do it later on. Note that the markdown file is generated and doesn't need to be updated, only the [mdoc](https://github.com/matrix-org/pantalaimon/blob/master/docs/man/pantalaimon.8) file

Reply:
I am very unfamiliar with mdoc. I am barely proficient in python.

Reply:
Ok, I'll open an issue about this so it doesn't get lost. Good to merge.